morebits.js

```diff
@@ -719,6 +726,55 @@ Morebits.quickForm.element.generateTooltip = function QuickFormElementGenerateTo
 // Some utility methods for manipulating quickForms after their creation:
 // (None of these work for "dyninput" type fields at present)
 
+/**
+ * Returns an object containing all filled form data entered by the user, with the object
+ * keys being the form element names. Disabled fields will be ignored, but not hidden fields.
```

Comment:
I was following the standard of any HTML form attached to a GET/POST endpoint, in which if you hit submit, the hidden field values are sent to the server, but disabled fields aren't. 

I haven't looked at `block` and `speedy` do, but I suppose one way is to fix them instead, seeing as the current behaviour is the standard?

Reply:
>I suppose one way is to fix them

Yes, I agree; was trying to think up issues.  They're not *too* egregious IIRC, but both rely on `hide`ing elements when switching modes rather than rebuilding them.  I think block will be tricky mainly because of how it handles the form, but neither are particularly urgent!

Reply:
Not important for this, but just copying here a list of modules not touched:

1. twinklearv.js
42. twinkleblock.js
13. twinkleconfig.js
42. twinklediff.js
13. twinklefluff.js
42. twinklespeedy.js